comp/core/autodiscovery/listeners/common.go

```diff
@@ -8,11 +8,12 @@
 package listeners
 
 import (
-	"fmt"
 	"hash/fnv"
 	"maps"
 	"strconv"
 
+	"github.com/samber/lo"
```

Comment:
I'm ok with using this dependency but I according to the CI it's now required in some agents where it wasn't required before. Not sure if this is something that needs to be avoided.

Reply:
I thought that `samber/lo` was already used because I didn’t have to add it to the `go.mod` file, but it was indeed not used by every binary.
I just pushed a commit to remove it: [`fd9fde7` (#44036)](https://github.com/DataDog/datadog-agent/pull/44036/commits/fd9fde7ff0d95d6bcf0a99327a9a14353d1e895b).